app/assets/javascripts/listings/components/listingContainer.js.coffee

```diff
@@ -25,14 +25,21 @@ angular.module('dahlia.components')
       @lotteryResultsListings = ListingService.lotteryResultsListings
 
       @isOwnershipListing = (listing) ->
-        @listing.Tenure == 'New sale' || @listing.Tenure == 'Resale'
+        listing.Tenure == 'New sale' || listing.Tenure == 'Resale'
```

Comment:
I made this update to be consistent with where we were going with moving towards passing listing to functions (it also makes it useable in the functions below)

Reply:
We should consider moving this one to ListingDataService when it gets merged with refactoring.